README.md

```diff
@@ -15,6 +15,8 @@ The Mobile Center SDK uses a modular architecture so you can use any or all of t
 
 3. **Mobile Center Distribute**: Mobile Center Distribute will let your users install a new version of the app when you distribute it via the Mobile Center. With a new version of the app available, the SDK will present an update dialog to the users to either download or postpone the new version. Once they choose to update, the SDK will start to update your application. This feature will NOT work if your app is deployed to the app store.
 
+4. **Mobile Center Push**: Mobile Center Push enables you to send push notifications to users of your app from Mobile Center portal. Mobile Center SDK and portal integrate different Push Notification Service (PNS) on different platforms - [Windows Notification Services](https://docs.microsoft.com/en-us/windows/uwp/controls-and-patterns/tiles-and-notifications-windows-push-notification-services--wns--overview) (WNS) for UWP, [Firebase Cloud Messaging](https://firebase.google.com/docs/cloud-messaging/) for Xamarin.Android and [Apple Push Notification Service](https://developer.apple.com/library/content/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/APNSOverview.html) for Xamarin.iOS.
```

Comment:
It's too early to say we support Xamarin for Push. Please say we support only UWP/WNS right now.